### ISSUE TITLE
Fix Fn::ImportValue validation

### ISSUE THREAD

**Issue by mwpolcik** [Appliscale/perun#160]
template example:
```yaml
AWSTemplateFormatVersion: '2010-09-09'

Resources:

  MyEC2:
    Type: "AWS::EC2::Instance"
    Properties:
      ImageId: ami-1111111
      InstanceType: t2.medium
      SubnetId: subnet-1231231
      KeyName: something
      SecurityGroupIds:
      - Fn::ImportValue: some-export-id
      IamInstanceProfile: some-profile
```
Issue:
```
[1] % perun validate test.yml
INFO: Configuration file from the following location will be used: /Users/mpolcik16/.config/perun/main.yaml
INFO: Creating new session. Profile: aol Region: us-east-1
WARNING: MyEC2: SecurityGroupIds[0] <--- is nil.
```
Perun is not respecting `Fn::ImportValue` and it's deciding it is nil, when in fact it is a valid value.

Expected behaviour: validation should pass.

**Comment by maxiwoj** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Fix Fn::ImportValue validation](https://trello.com/c/X90KQzVs/62-fix-fnimportvalue-validation)

**Comment by maxiwoj** [comment]
closed by #174
